Apps/Admin/Client/Pages/FeedbackPage.razor

```diff
@@ -129,21 +131,28 @@ else
                 </MudTd>
                 <MudTd DataLabel="Comments" data-testid="feedback-comments">@context.Comments</MudTd>
                 <MudTd DataLabel="Tags">
-                    <MudSelect T="Guid"
-                               MultiSelection="true"
-                               SelectedValues="@context.TagIds"
-                               SelectedValuesChanged="@(tagIds => AssociateTags(tagIds, context.Id))"
-                               MultiSelectionTextFunc="@DescribeTags"
-                               Disabled="@FeedbackUpdating"
-                               Variant="@Variant.Filled"
-                               Margin="@Margin.Dense"
-                               Dense="true"
-                               data-testid="feedback-tag-select">
-                        @foreach (AdminTagView tag in Tags)
-                        {
-                            <MudSelectItem T="Guid" Value="@tag.Id" data-testid="feedback-tag">@tag.Name</MudSelectItem>
-                        }
-                    </MudSelect>
+                    <div class="d-flex justify-end">
+                        <MudSelect T="Guid"
+                                   MultiSelection="true"
+                                   SelectedValues="@context.TagIds"
+                                   SelectedValuesChanged="@(tagIds => AssociateTags(tagIds, context.Id))"
+                                   MultiSelectionTextFunc="@DescribeTags"
+                                   Disabled="@FeedbackUpdating"
+                                   Variant="@Variant.Filled"
+                                   Margin="@Margin.Dense"
+                                   Dense="true"
+                                   data-testid="feedback-tag-select">
+                            @foreach (AdminTagView tag in Tags)
+                            {
+                                <MudSelectItem T="Guid" Value="@tag.Id" data-testid="feedback-tag">@tag.Name</MudSelectItem>
+                            }
+                        </MudSelect>
+                        <MudIconButton OnClick="@(() => SaveAssociatedTags(context.Id))"
+                                       Icon="fas fa-save"
+                                       Color="@Color.Primary"
+                                       Disabled="@(!context.IsDirty)"
```

Comment:
I think this should also be disabled when FeedbackUpdating is true.

(I was going to make a comment about setting the "Loading" property as well, but unfortunately, we don't yet have an HgIconButton custom component that supports it the same way we do for HgButton.)